### PR TITLE
Add Maze icon

### LINKED ISSUE #10925: Request: Maze
### Brand Name

Maze

### Website

https://maze.co

### Popularity Metric

The similar web rank is 69,606. See: https://www.similarweb.com/website/maze.co/#overview

### Official Resources for Icon and Color

None

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by RaghavMaliwal**
image

**Issue:** closes #10925

**Popularity metric:**

The similar web rank is 69,606. See: https://www.similarweb.com/website/maze.co/#overview

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by adamrusted** [comment]
See their [terms of service](https://maze.co/tos/#intellectual-property-rights), stating that they do not give permission for their logo to be used.  Could you please reach out to them to clarify the use-case @RaghavMaliwal? You can find info about requesting permission [in our guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-permission).
image

**Comment by RaghavMaliwal** [comment]
@adamrusted Can you please tell whom to mail to ask for the permission...!

**Comment by RaghavMaliwal** [comment]
Okay...

**Comment by RaghavMaliwal** [comment]
@adamrusted I have submitted a request on their Contact Us page. Let's wait for their response
Screenshot 2024-05-24 135835

**Comment by RaghavMaliwal** [comment]
Hello @adamrusted, Received a positive response from their side...
image

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json [hunk 1]
```diff
@@ -9345,6 +9345,11 @@
             "hex": "101010",
             "source": "https://www.mazda.com/en/about/profile/library/"
         },
+        {
+            "title": "Maze",
+            "hex": "000000",
+            "source": "https://maze.co"
```
Comment (uncenter) [inline _data/simple-icons.json:9351]: Press kit linked in the footer.
```suggestion
            "source": "https://mazedesign.notion.site/Press-Kit-421d405e76154662a547ce9ef2a3f018"
```

File: icons/maze.svg
[the remark this answers is not in the document]
Comment (uncenter) [reply icons/maze.svg:1]: Still seeing these hairline differences on the top and bottom. Make sure you use the SVG from the press kit source - your icon is a bit taller than my normalized version of the original for whatever reason, so maybe try recreating from scratch.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/maze.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Maze</title><path d="M16.07776 5.99642c-.9839 0-1.96769.37216-2.71839 1.11668L5.2038 15.20191c-.7507.74452-1.96809.74415-2.71878 0-.75068-.74452-.75069-1.95161 0-2.69613L6.5628 8.46117c.51455-.51032 1.2484-.67078 1.90081-.48153.50948.14779 1.12191.17996 1.49727-.1923.37536-.37226.37941-.98925-.07695-1.25817-1.47537-.86939-3.41094-.6748-4.68014.58394l-4.07777 4.0442c-1.50137 1.48902-1.50135 3.90321 0 5.39227 1.50138 1.48908 3.93536 1.48945 5.43678 0l8.15557-8.08841c.7507-.7445 1.96807-.74451 2.71878 0l4.07778 4.0442c.7506.74452.7506 1.95161 0 2.69613-.75081.7445-1.96809.7445-2.71878 0l-1.40782-1.39614c-.72388-.71792-1.89723-.71792-2.62112 0l-1.74767 1.73345c-.51454.51032-1.24839.67078-1.9008.48153-.50948-.14778-1.12191-.17995-1.49728.1923-.37534.37226-.37941.98925.07695 1.25817 1.47537.86939 3.41094.6748 4.68013-.58394l1.69924-1.68538 1.35939 1.34806c1.50139 1.48904 3.93536 1.48904 5.43678 0 1.50141-1.48902 1.50141-3.90325 0-5.39227l-4.0778-4.04418c-.7507-.74451-1.73449-1.11668-2.71839-1.11668z"/></svg>
```
Comment (uncenter) [inline icons/maze.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Maze</title><path d="M1.126 16.547c-1.5013-1.4881-1.5013-3.9009 0-5.389l4.0778-4.042c1.2692-1.258 3.205-1.4525 4.6803-.5836.4564.2687.4524.8852.077 1.2573-.3753.372-.988.34-1.4975.1923-.6524-.1891-1.386-.0287-1.9006.4813l-4.0777 4.0419a1.8935 1.8935 0 0 0 0 2.6945c.7506.744 1.9678.744 2.7184 0l8.1555-8.0836c1.5014-1.4882 3.9355-1.4882 5.437 0l4.0778 4.0418c1.5013 1.4881 1.5013 3.901 0 5.389-1.5014 1.4882-3.9356 1.4882-5.437 0l-1.3593-1.3472-1.699 1.684c-1.2692 1.258-3.205 1.4526-4.6804.5837-.4563-.2687-.4523-.8852-.077-1.2573.3754-.372.988-.34 1.4975-.1923.6524.1892 1.386.0287 1.9006-.4813l1.7476-1.7322c.724-.7175 1.8975-.7175 2.6214 0l1.4078 1.3954c.7507.744 1.9678.744 2.7186 0a1.8936 1.8936 0 0 0 0-2.6945l-4.0779-4.0419c-.7507-.744-1.9678-.744-2.7185 0L6.563 16.5471c-1.5014 1.4882-3.9356 1.4881-5.437 0"/></svg>
```

Comment (mondeja) [reply icons/maze.svg:1]: Yeap, this solves the problem.

**Review by mondeja — APPROVED** [review]
Thanks both for the great work!

**Pull request merged**